reana_server/config.py

```diff
@@ -70,8 +71,16 @@ def _(x):
 ACCOUNTS_USERINFO_HEADERS = True
 #: Disable password recovery by users.
 SECURITY_RECOVERABLE = False
-#: Disable user to confirm their email address.
-SECURITY_CONFIRMABLE = False
+REANA_USER_CONFIRMATION = strtobool(os.getenv("REANA_USER_CONFIRMATION", "true"))
```

Comment:
What do you think about leaving some comment above this line? Something like: `#: User's email address confirmation` could be used? Because it could be thought that it's related with `SECURITY_RECOVERABLE`